standalone/node/src/chain_spec.rs

```diff
@@ -222,6 +222,72 @@ pub fn local_testnet_config() -> Result<ChainSpec, String> {
 	))
 }
 
+pub fn relayer_testnet_config() -> Result<ChainSpec, String> {
+	let wasm_binary = WASM_BINARY.ok_or_else(|| "Development wasm not available".to_string())?;
+	let mut properties = sc_chain_spec::Properties::new();
+	properties.insert("tokenSymbol".into(), "tTNT".into());
+	properties.insert("tokenDecimals".into(), 18u32.into());
+	properties.insert("ss58Format".into(), 42.into());
+
+	Ok(ChainSpec::from_genesis(
+		// Name
+		"Local Testnet",
+		// ID
+		"local_testnet",
+		ChainType::Local,
+		move || {
+			relayer_testnet_genesis(
```

Comment:
Any reason behind why we have `relayer_testnet_genesis` and not using the OG `testnet_genesis`? what is the difference between both?

Reply:
I would keep using the same for now, if we found ourselves tweaking it a lot, we can diverge and create a separate function.

Reply:
I did revert but I want just 2 initial authorities in chain_spec. I will keep relayer_test_config to generate chain_spec using the same `testnet_genesis`

Reply:
Here is an example of something we use in DKG to control the Initial authorities count using an env https://github.com/webb-tools/dkg-substrate/blob/0d8321cc19f3f38603d42992401710f2a507de27/standalone/node/src/chain_spec.rs#L169-L178

You can do the same without introducing a new genesis config.

Reply:
We use it here https://github.com/webb-tools/dkg-substrate/blob/0d8321cc19f3f38603d42992401710f2a507de27/standalone/node/src/chain_spec.rs#L189-L193
and here: https://github.com/webb-tools/dkg-substrate/blob/0d8321cc19f3f38603d42992401710f2a507de27/standalone/node/src/chain_spec.rs#L197-L204

Reply:
Well actually, never mind, since this happens during the build-time not runtime. forget what I said above, adding a chainspec for the relayer is better.